### PR TITLE
BaSP-MR-44: Admin LIST

### PR DESCRIPTION
**Pull request by DelfiLT**
Create a component wich includes the following functionality:

- Add item to the list
- Delete item from the list
- Update item from the list
+ confirm and server messages

### CODE REVIEW

**Review by Dannylez — APPROVED** [review]
seems good to me 🚀

File: src/Components/Admins/Button.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const Button = ({ onclick, setStyle, text }) => {
```
Comment (Juanoauil) [inline src/Components/Admins/Button.js:1]: component files should be .jsx

**Review by SantiH97 — APPROVED** [review]
Not only do I have nothing to suggest be changed but I also took note of some of your implementations to make my code a little better. Keep up the good work!!

File: src/Components/Admins/index.js [hunk 2]
```diff
@@ -1,9 +1,113 @@
...
+        method: 'PUT',
+        headers: {
+          'Content-type': 'application/json'
+        },
+        body: JSON.stringify(updatedAdmin)
+      });
+
+      const { message, data, error } = await res.json();
+      alert(message);
+      console.log(data);
```
Comment (Juanoauil) [inline src/Components/Admins/index.js:73]: remove console.log

File: src/Components/Admins/index.js [hunk 3]
```diff
@@ -1,9 +1,113 @@
...
+      });
+
+      const { message, data, error } = await res.json();
+      alert(message);
+
+      if (!error) {
+        setAdmins([...admins, data]);
+      }
+    } catch (error) {
+      console.log(error);
```
Comment (Juanoauil) [inline src/Components/Admins/index.js:56]: it should be console.error

File: src/Components/Admins/index.js [hunk 4]
```diff
@@ -1,9 +1,113 @@
...
+      const { message, data, error } = await res.json();
+      alert(message);
+      console.log(data);
+      if (!error) {
+        const actualAdmins = [...admins];
+        actualAdmins[adminIndex] = data;
+        setAdmins(actualAdmins);
+      }
+    } catch (error) {
+      console.log(error);
```
Comment (Juanoauil) [inline src/Components/Admins/index.js:80]: same here

File: src/Components/Admins/index.js [hunk 5]
```diff
@@ -1,9 +1,113 @@
...
+          method: 'DELETE'
+        });
+
+        const data = await res.json();
+        alert(data.message);
+
+        setAdmins(admins.filter((admin) => admin._id !== id));
+      }
+    } catch (error) {
+      console.log(error);
```
Comment (Juanoauil) [inline src/Components/Admins/index.js:98]: same here

File: src/Components/Admins/index.js (hunk 3 above)
Comment (Juanoauil) [reply src/Components/Admins/index.js:56]: implement this in all of console.log

**Review by Juanoauil — APPROVED** [review]

**Pull request merged**